dashboard/test/controllers/api/v1/census/census_controller_test.rb

```diff
@@ -93,6 +93,45 @@ class Api::V1::Census::CensusControllerTest < ActionController::TestCase
     refute response['census_submission_id'].nil?, "census_submission_id expected in response: #{response}"
   end
 
+  test 'yourschool census submission with negative 1 school_name, existing school_id succeeds' do
```

Comment:
nit: what does the negative 1 part of this symbolize?

Reply:
"negative 1" indicates that the school was not found in the dropdown menu and instead, the user checked the school not found box